JASP-Desktop/analysis/options/term.cpp

```diff
@@ -20,20 +20,21 @@
 #include "utilities/qutils.h"
 #include <sstream>
 
+const char * Term::separator =
+#ifdef _WIN32
+		" * ";
```

Comment:
Why do we have stars on windows and some magical symbols on unix?

Reply:
It was like that (I just moved this code). I suppose that on Mac and Linux, the '*' sign was not nicely displayed with the fonts they used.

Reply:
Yeah, ok, I vaugely remember reading someone request your help on changing that `*` so I wondered if you knew why, but it doesn't really matter anyway, because it doesn't change anything.